adafruit_featherwing/dotstar_featherwing.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, clock=board.D13, data=board.D11, brightness=0.2):
         self.rows = 6
         self.columns = 12
         self._auto_write = True
-        self._dotstar = dotstar.DotStar(clock, data, self.rows * self.columns,
+        self._display = dotstar.DotStar(clock, data, self.rows * self.columns,
```

Comment:
I would consider naming this `matrix` instead of `display` as we have a lot of display libraries within which we use that term. There are some FeatherWings with displays (or maybe only one?) and it might be a little confusing.